packages/shared/routes/onboarding/views/CongratulationsView.svelte

```diff
@@ -21,7 +21,7 @@
 
     function _continue(): Promise<void> {
         completeOnboardingProcess()
-        $onboardingRouter.next()
+        $onboardingRouter.next({ ledgerRecovery: true })
```

Comment:
What is the reason for including `{ ledgerRecovery: true }`? I can't find anywhere else in the code that needs this and the ledger profile recovery still works with it removed.

Reply:
This was a mistake i was trying something else originally let me update that

Reply:
Done